examples/solo8_vanilla/observation_dump.py

```diff
@@ -0,0 +1,36 @@
+""" A demo for the Solo v2 Vanilla with configurable observations
+
+The motors have no torque applied to them; the user can interact with the robot
+via the interactive GUI. The simulation will output the observation.
+"""
+import argparse
+import gym
+import numpy as np
+
+import gym_solo
+from gym_solo.envs import solo8v2vanilla
+from gym_solo.core import obs
+
+
+if __name__ == '__main__':
+  config = solo8v2vanilla.Solo8VanillaConfig()
+  env = gym.make('solo8vanilla-v0', use_gui=True, realtime=True, config=config)
+
+  env.obs_factory.register_observation(obs.TorsoIMU(env.robot))
+
+  try:
+    print("""\n
+          =============================================
+              Solo 8 v2 Vanilla Observation Dump
+              
+          Simulation Active.
+          
+          Exit with ^C.
+          =============================================
+          """)
+
+    while True:
+      obs, reward, done, info = env.step(np.zeros(env.action_space.shape))
+      print(obs)
```

Comment:
Do we need print statements?

Reply:
Oh, I kept that in purely because it is an example file and it's supposed to output the current observations. I can change it, but do you have any recommendations for displaying the observation to the user?

Reply:
I see your point! In that case, looks good to me!